crates/python/qcs_sdk/qvm/api.pyi

```diff
@@ -0,0 +1,230 @@
+from typing import Dict, List, Sequence, Mapping, Optional, Tuple, final
+from typing_extensions import Self
+
+from qcs_sdk import RegisterData
+from qcs_sdk.client import QCSClient
+
+def get_version_info(client: Optional[QCSClient] = None) -> str:
```

Comment:
I'd been writing this as `client: Optional[QCSClient] = ...` trying to indicate that a client is actually constructed, not sure it's actually better but curious if you had a particular opinion on one vs the other

Reply:
My only qualm with using `...` is that it doesn't faithfully represent the actual default value. `...` is its own type in Python, and if someone were to explicitly call `get_version_info(client=...)`, they would get an error.

Reply:
I've gone ahead and replaced all the `...` with `None` just so they are consistent, but happy to revisit the topic if needed.